spec/system/admin/admin_views_tags_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
```

Comment:
There's probably a better way to write this test, but IMO this is a good enough integration test that will probably be rewritten in Cypress.

Reply:
hello, can we reuse the icon we have on the Mod Panel?

https://share.getcloudapp.com/rRu0xmBp